openlibrary/macros/EditionNavBar.html

```diff
@@ -1,31 +1,38 @@
-$def with (reader_observations, edition_count=1, show_observations=False)
+$def with (reader_observations, edition_count=1, show_observations=False, desktop_only=False)
```

Comment:
Was confused by `desktop_only` -- only implies 2 things can't happen concurrently, what are those 2 things here?

Would recommend either `desktop_mode=False` or `more=desktop`, `mode=mobile`.

Anyway, the `_only` threw me off